src/components/Speedometer.tsx

```diff
@@ -9,6 +9,11 @@ interface IProps {}
 interface IState {
     gw2data_ipc: IGw2MumbleLinkData | null;
     gw2data_ws: IGw2MumbleLinkData | null;
+    now_position: any;

```

Comment:
Would be nice if you had type here, it's an interface IPosition that has this structure.

That would also highlight a bug on line 28 where you use small xyz instead of capital XYZ